README.md

```diff
@@ -1,186 +1,246 @@
+information, head to the [CONTRIBUTING.md] file.
 
-Thank you to all our backers! [Become a backer](https://opencollective.com/poliastro#backer).
+Release announcements and general discussion take place on our [mailing list].
 
-[![Backers](https://opencollective.com/poliastro/backers.svg?width=890)](https://opencollective.com/poliastro#backer)
-
-## Sponsors
-
-Support us by becoming a sponsor. Your logo will show up here with a link to your website.
-[Become a sponsor](https://opencollective.com/poliastro#sponsor).
+For further clarifications and discussions, feel free to join poliastro's [chat
+room].
 
-[![Sponsors](https://opencollective.com/poliastro/sponsor/0/avatar.svg)](https://opencollective.com/poliastro/sponsor/0/website)
+![Contributors image](https://opencollective.com/poliastro/contributors.svg?width=890&button=false)
 
-# FAQ
 
-## What's up with the name?
+## Backers and sponsors
 
-poliastro comes from Polimi, which is the shortened name of the
-Politecnico di Milano, the Italian university where I was studying while
-writing this software. It's my tiny tribute to a place I came to love.
-*Grazie mille!*
+poliastro requires finnacial support to mantain its high quality standars. The
+money is used to renew the web domain and updating the documentation hosting
+subscription among others.
 
-## Is poliastro validated?
+If you would like to support poliastro, consider [becoming a backer] or
+[becoming a sponsor]. 
 
-Yes! poliastro is a community project that strives to be easy to use,
-while at the same time producing correct results
-[that are validated](https://github.com/poliastro/validation/)
-against other [commonly used Astrodynamics software](https://docs.poliastro.space/en/latest/related.html)
-such as GMAT and Orekit.
+**Thanks to all our backers!**
 
-## Can I suggest new features for poliastro?
+[![Backers](https://opencollective.com/poliastro/backers.svg?width=890)](https://opencollective.com/poliastro#backer)
 
-Sure, we encourage you to [open an issue](https://github.com/poliastro/validation/issues/new)
-so we can discuss future feature additions!
 
-## What's the future of the project?
+**Thanks to all our sponsors!**
 
-poliastro is actively maintained and receiving an influx of new
-contributors thanks to the generous sponsorship of Google and the
-European Space Agency. The best way to get an idea of the roadmap is to
-see the [Milestones](https://github.com/poliastro/poliastro/milestones)
-of the project.
+[![Sponsors](https://opencollective.com/poliastro/sponsor/0/avatar.svg)](https://opencollective.com/poliastro/sponsor/0/website)
+[![Sponsors](https://opencollective.com/poliastro/sponsor/1/avatar.svg)](https://opencollective.com/poliastro/sponsor/0/website)
+
+
+
+
+## Frequently asked questions
+
+* **What's up with the name?**
+
+  poliastro comes from Polimi, which is the shortened name of the
+  Politecnico di Milano, the Italian university where I was studying while
```

Comment:
Maybe this paragraph could stop there, and add "see more at https://docs.poliastro.space/en/stable/history.html#acknowledgement-from-the-original-author" to avoid the first person singular here